namerd/storage/consul/src/main/scala/io/buoyant/namerd/storage/consul/ConsulDtabStore.scala

```diff
@@ -212,6 +225,51 @@ class ConsulDtabStore(
         Future.Unit
       }
     }
-    Activity(run).stabilize
+    dtabStatus.putIfAbsent(ns, InstrumentedDtab(run, pollState))
+    Activity(run.underlying).stabilize
   }
+
+  val handlerPrefix = root.show.drop(1) // drop leading "/"
+
+  override def adminHandlers: Seq[Admin.Handler] = Seq(
+    Admin.Handler(
+      s"/namer_dtabs_store/${handlerPrefix}.json",
```

Comment:
how about `storage` for the first segment?  so that this becomes `/storage/dtabs.json`?

Reply:
I am currently working on the k8s version of this implementation and was thinking about how the endpoint would look like in the k8s scenario. I like the `/storage/dtab.json`. What about including the storage type as well in the link like `/storage/io.l5d.consul/dtabs.json` or `/storage/consul/dtabs.json`?

Reply:
including the type is less important here than it is for namers or interpreters because there can only be one storage module configured in Namerd (whereas there may be many namers configured).

Reply:
Ahh I see, I was under the impression that you could have multiple storage modules in namerd.

Reply:
fine for me